### ISSUE TITLE
Error with 1.20.6

### ISSUE THREAD

**Issue by SkytAsul** [MockBukkit/MockBukkit#1006]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest version of MockBukkit?

- [X] I am using the latest version of MockBukkit.

### Minecraft Version

1.20

### Describe the bug

When mocking a server and a player on 1.20.6, this error happens:
```
java.lang.NoSuchFieldError: Class org.bukkit.attribute.Attribute does not have member field 'org.bukkit.attribute.Attribute HORSE_JUMP_STRENGTH'
	at be.seeseemelk.mockbukkit.attribute.AttributesMock.<clinit>(AttributesMock.java:27)
	at be.seeseemelk.mockbukkit.entity.LivingEntityMock.<init>(LivingEntityMock.java:119)
	at be.seeseemelk.mockbukkit.entity.HumanEntityMock.<init>(HumanEntityMock.java:74)
	at be.seeseemelk.mockbukkit.entity.PlayerMock.<init>(PlayerMock.java:220)
	at be.seeseemelk.mockbukkit.entity.PlayerMockFactory.createRandomPlayer(PlayerMockFactory.java:59)
	at be.seeseemelk.mockbukkit.ServerMock.addPlayer(ServerMock.java:343)
```

### Reproducible Test

```java
MockBukkit.mock().addPlayer();
```
With MockBukkit-v1.20:3.86.0

**Comment by Insprill** [comment]
1.20.5/6 is not supported yet.

**Comment by Thorinwasher** [comment]
Fixed in #1008
